### PR TITLE
[wiki] Add Initial steps of commissioning Thread device via OTBR snap 

### PR DESCRIPTION
**Pull request by MonicaisHer**
The purpose of this PR is to facilitate easier review and feedback on the content wiki page. The content of this PR will eventually be moved to the wiki page of this repository but will not be merged into the main branch.

### CODE REVIEW

File: Wiki.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+> dataset init new
+Done
+> dataset commit active
+Done
+> ifconfig up
+Done
+> thread start
+Done
+```
+Alternatively, these steps could also be performed in the GUI at http://0.0.0.0/.
```
Comment (farshidtz) [inline Wiki.md:48]: The loopback interface address should be localhost or 127.0.0.1. Adresse 0.0.0.0 may work on some systems but it isn't usually used for this purpose.

File: Wiki.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+> ifconfig up
+Done
+> thread start
+Done
+```
+Alternatively, these steps could also be performed in the GUI at http://0.0.0.0/. 
+Please refer to the instructions [here](https://openthread.io/guides/border-router/web-gui.md) to configure and form, join, or check the status of a Thread network using GUI.
+
+## Discovering and pairing the Thread lighting device into the OTBR network
+Make sure the Thread lighting device is running. Flashing and running the Thread lighting device is beyond the scope of this guide.
```
Comment (farshidtz) [inline Wiki.md:52]:
```suggestion
Make sure the Thread lighting app is running. Flashing and running the Thread lighting app on the device is beyond the scope of this guide. 
```

**Review by farshidtz — CHANGES_REQUESTED** [review]
A simple and useful guide. I've added some minor comments.

**Comment by MonicaisHer** [comment]
Close this PR, as the content has been moved to [wiki](https://github.com/MonicaisHer/openthread-border-router-snap/wiki/Commission-and-control-a-Matter-Thread-device-via-the-OTBR-Snap) page.

**Pull request closed without merging**